engine/sync_manager.go

```diff
@@ -697,12 +697,12 @@ func printCurrencyFormat(price float64, displayCurrency currency.Code) string {
 }
 
 func printConvertCurrencyFormat(origPrice float64, origCurrency, displayCurrency currency.Code) string {
-	var conv float64
+	var conv string
 	if origPrice > 0 {
-		var err error
-		conv, err = currency.ConvertFiat(origPrice, origCurrency, displayCurrency)
-		if err != nil {
-			log.Errorf(log.SyncMgr, "Failed to convert currency: %s", err)
+		if convFloat, err := currency.ConvertFiat(origPrice, origCurrency, displayCurrency); err != nil {
+			conv = "?.??"
+		} else {
+			conv = fmt.Sprintf("%.2f", convFloat)
```

Comment:
Amounts get dropped, this allows it to reinstate it. 
```diff
diff --git a/engine/sync_manager.go b/engine/sync_manager.go
index b8afaf602..22db70952 100644
--- a/engine/sync_manager.go
+++ b/engine/sync_manager.go
@@ -704,6 +704,8 @@ func printConvertCurrencyFormat(origPrice float64, origCurrency, displayCurrency
                } else {
                        conv = fmt.Sprintf("%.2f", convFloat)
                }
+       } else {
+               conv = "0.00"
        }

        displaySymbol, err := currency.GetSymbolByCurrencyName(displayCurrency)
```

Reply:
Yep. Thanks.

Fixed f22a191b